src/ucm/api/ucm.h

```diff
@@ -323,6 +323,21 @@ void ucm_unset_external_event(int events);
 ucs_status_t ucm_test_events(int events);
 
 
+/**
+ * @brief Test event external handlers
+ *
+ * This routine checks if event handlers are called when corresponding system API
+ * is invoked through external infrastructure, see @ref ucm_set_external_event
+ * for details.
```

Comment:
`This routine checks if external events, as set by @ref ucm_set_external_event, are actually being reported (by calling APIs such as @ref ucm_vm_munmap)`